src/ui/articlesaver.hh

```diff
@@ -41,12 +42,28 @@ class Class;
 class QWidget;
 
 namespace ArticleSaver {
```

Comment:
namespace is a good pratice.

But IMO ,we can remove the namespace right now.   to keep consistent with other class file.